winforms/src/toga_winforms/libs/proactor.py

```diff
@@ -142,7 +142,15 @@ def run_once_recurring(self):
             # to be processed. If you don't ensure there is at least one
             # message on the queue, the select() call will block, locking
             # the app.
```

Comment:
Given the logic here has changed, updating the comment would also seem appropriate.